### DISCUSSION TITLE
Data security

### DISCUSSION THREAD

**Discussion by VivekMParekh** [tvtoglu/predictr#8] [Q&A]
Is this library ensure security of data being analyzed under the Analysis function? Is it secure from data-security perspective?

**Comment by tvtoglu** [comment]
I am not sure what you mean by data security in the context of this library. However, let me answer your question like this:
No data is being transferred to a server or is accessible outside your local system. 
Does that answer your question? Let me know. 

Best regards

**Reply by VivekMParekh** [reply]
Yes, thank you. Congratulation for great work.
